lib/segment/src/common/reciprocal_rank_fusion.rs

```diff
@@ -24,7 +24,8 @@ fn position_score(position: usize) -> f32 {
 /// Does not break ties.
 pub fn rrf_scoring(
     responses: impl IntoIterator<Item = Vec<ScoredPoint>>,
-    limit: usize,
+    take: usize,
```

Comment:
Might be better to separate rescoring from skip and limit and return iterator instead